### PR TITLE
job description for interns

### PR DESCRIPTION
**Pull request by nicksnyder**

### CODE REVIEW

File: job-descriptions/software-engineer-intern.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### About the team
+
+Our small team consists of talented, mature, collaborative, driven individuals who are attracted to the massive problem we are tackling. We work in an open environment that treats people in a first-class manner and provides them with ownership, responsibility, and autonomy.
+
+### About the role
+
+You will:
+
+- Be paried with a mentor who will help you plan your work and teach you what you need to know.
```
Comment (dadlerj) [inline job-descriptions/software-engineer-intern.md]: "paried" -> "paired"

File: job-descriptions/software-engineer-intern.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+We have already built the world's best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large and small engineering organizations across the world.
+[Sourcegraph](https://about.sourcegraph.com/product/server) and our [browser extensions](https://about.sourcegraph.com/product/browser) provide IDE-like code intelligence (e.g. hover tooltips, go to definition, find references, and much more) in your code host, in code review, and everywhere you read code.
+
+Read [our blog](https://about.sourcegraph.com/blog/) to see what more we have been working on recently and read [our master plan](https://sourcegraph.com/plan) to see where we are going!
+
+### About the team
+
+Our small team consists of talented, mature, collaborative, driven individuals who are attracted to the massive problem we are tackling. We work in an open environment that treats people in a first-class manner and provides them with ownership, responsibility, and autonomy.
+
+### About the role
```
Comment (dadlerj) [inline job-descriptions/software-engineer-intern.md]: Can we add a few "Examples of projects that interns have worked on in the past"? I think that's the most powerful way to communicate the scope. E.g.:

> Example Software Engineer Intern projects from past years:
> - Build a browser extension from scratch that communicates with the Sourcegraph API to inject code intelligence on GitHub.com.
> - ...

Comment (nicksnyder) [reply job-descriptions/software-engineer-intern.md]: Yeah, I am happy to add examples. All of our previous interns predate me starting so I don't know what all the projects were. If you know can you share?

Comment (dadlerj) [reply job-descriptions/software-engineer-intern.md]: Great question — @attfarhan is my description of yours above correct? And @ggilmore can you share yours? And what did the other interns (Julian? Joe? Shantanu?) do?

Comment (attfarhan) [reply job-descriptions/software-engineer-intern.md]: Yup that's correct.

Julien, Rohan, and I all spent some time working on the Sourcegraph Desktop app, which was an electron app + editor extension combo. The electron app had the Sourcegraph web view, and would update as you changed positions in your editor.

Rohan also spent a week or so on a Sourcegraph search integration for Alfred, which was pretty cool.

**Pull request merged**